docs/data/charts/legend/legend.md

```diff
@@ -71,6 +71,18 @@ For the `pie` series, the `labelMarkType` property is available for each of the
 
 {{"demo": "LegendMarkType.js", "hideToolbar": true, "bg": "playground"}}
 
+#### Custom shapes
+
+For more advanced use cases, you can also provide a component to the `labelMarkType` property of each series to fully customize the mark.
+
+{{"demo": "LegendCustomLabelMark.js" }}
+
+Passing a component to `labelMarkType` affects not only the legend but other places where the label mark is shown, such as tooltips.
+
+Customizing the mark shape of a pie chart depending on the series is slightly different. You can find how to do it in [this example](/x/react-charts/pie-demo/#pie-chart-with-custom-mark-in-legend-and-tooltip).
+
+Please note that using HTML elements for `labelMarkType` may make the legend incompatible with [gradients and patterns](/x/react-charts/styling/#gradients-and-patterns).
```

Comment:
I would word it more positively like:

```suggestion
To ensure compatibility with [gradients and patterns](/x/react-charts/styling/#gradients-and-patterns), consider using SVG instead of HTML in the `labelMarkType`.
```